_posts/2015-12-15-promise-again

```diff
@@ -0,0 +1,11 @@
+---
+layout: post
+title:  "Promise And Resolve"
+date:   2015-12-15 01:49:56
+categories: nodejs javascript
+author:
+ twitter: sahilbathla
+ name: Sahil Batla
+---
+
+Sirf "Promise" mat karo, Use "Resolve" bhi karo
```

Comment:
Use this for writing text in hindi
http://www.quillpad.in/index.html#.VmfyPBArKRs